lms/validation/_bearer_token.py

```diff
@@ -60,6 +60,7 @@ class BearerTokenSchema(marshmallow.Schema):
 
     user_id = marshmallow.fields.Str(required=True)
     oauth_consumer_key = marshmallow.fields.Str(required=True)
+    roles = marshmallow.fields.Str(required=True)
```

Comment:
Adding `roles` to `BearerTokenSchema` ensures that:

1. When `request.lti_user` is serialized into an authorization param, the user's roles are now serialized into that JWT as well
2. When an authorization param is deserialized into an `LTIUser` the roles are also deserialized